.github/renovate.json5

```diff
@@ -15,6 +15,16 @@
       "matchStrings": [
         " *(?<project>.*) version: (?<currentValue>.*)"
       ]
+    },
+    {
+      // Update minikube action parameters
+      "fileMatch": [ "^Dockerfile$" ],
+      "datasource": "github-tags",
+      "depName": "PowerDNS/pdns",
+      "currentValueTemplate": "auth-{{ver}}",
```

Comment:
Not sure if this will work. I'm fairly sure the `currentValue` will be handled well, but I don't know if renovate can handle prefixes in tags.

Reply:
Maybe related: https://github.com/renovatebot/renovate/issues/14546

Reply:
Let's try it this way. If not, `git-refs` manager does support it: https://docs.renovatebot.com/modules/datasource/git-refs/